3.11/aql/invocation-with-arangosh.md

```diff
@@ -328,6 +328,46 @@ Options related to the query optimizer.
   a rule, prefix its name with a `-`, to enable a rule, prefix it with a `+`. There is also
   a pseudo-rule `all`, which matches all optimizer rules. `-all` disables all rules.
 
+- `stream`: Specify `true` and the query is executed in a **streaming** fashion. The query result is
+  not stored on the server, but calculated on the fly. **Warning**: long-running queries
+  need to hold the collection locks for as long as the query cursor exists. It is advisable
+  to *only* use this option on short-running queries *or* without exclusive locks.
+  When set to `false`, the query is executed right away in its entirety.
+  In that case, the query results are either returned right away (if the result
+  set is small enough), or stored on the arangod instance and can be accessed
+  via the cursor API. 
+
+- `retriable`: Specify `true` and the request to retrieve the result from the latest batch fetched
```

Comment:
I have a hard time understanding the sentences in this paragraph. Can we shorten the individual sentences to make them easier to digest and more readable?

Reply:
I rewrote the descriptions.